ios/RNAdvertisingId.m

```diff
@@ -8,6 +8,11 @@
 
 @interface RCT_EXTERN_MODULE(RNAdvertisingId, NSObject)
 
-RCT_EXTERN_METHOD(getAdvertisingId:(RCTResponseSenderBlock)callback)
+RCT_EXTERN_METHOD(getAdvertisingId:(RCTPromiseResolveBlock)resolve rejecter: (RCTPromiseRejectBlock)reject)
```

Comment:
Here is the crux of the fix. I went to install this package on my react native application and I was getting `is not a recognized Objective-C method` errors on iOS. The swift and objective-c method signatures did not appear to align. Locally I fixed it by doing this and thought I would contribute it back.

After opening this repo locally I was unable to get the demo app running on Xcode 11 and iOS 13 which led to the other changes related to the swift version, bridging header files, package-lock.json file changes that came naturally when installing, and react native version update.